packages/react/src/components/SuggestionMenu/emojisMenu.tsx

```diff
@@ -0,0 +1,20 @@
+
+type Props= {
+  items: string[],
+  emojiInsert: (item : never) => void,
+  selectedIndex: number
+}
+  export default function EmojiMenu({items, emojiInsert, selectedIndex} : Props) : JSX.Element {
+  //this is the component which renders emoji picker with the emojis searched.
+
+
+    return (
+    <div style={{display: 'grid', justifyItems: 'center', background: '#1d1d1d', borderRadius: 20, gap: 7, padding: 20, maxHeight: '30vh', minWidth: '30vh', overflowY: 'scroll', gridTemplateColumns: 'repeat(10, 1fr)'}}>
```

Comment:
we should move these styles to CSS imo similar to how we do this for other components (cc @matthewlipski, what's the best approach for this?)

Reply:
added the styles in editor/styles.css as it had direct access to the CSS variables, which also helped in implementing dark mode.